main.py

```diff
@@ -1,13 +1,14 @@
-from logic import *
-
-#Start your program here
-
-a = 1
-
-def b():
-    global a
-    a += 2
-    print( " " + str(a))
-
-b()
-print(a)
\ No newline at end of file
+sum_list = []
+
+while True:
+    number_1 = int(input ("What is number 1"))
+    number_2 = int(input ("What is number 2"))
+    print(number_1 + number_2)
```

Comment:
Sometimes you can make your program look better by adding blank lines throughout at appropriate spots. Aesthetics are VERY important when you start writing larger programs.